projects/plugins/jetpack/class.jetpack-plan.php

```diff
@@ -2,39 +2,45 @@
 /**
  * Handles fetching of the site's plan and products from WordPress.com and caching values locally.
  *
+ * @deprecated $$next-version$$ use Automattic\Jetpack\Current_Plan instead.
+ *
  * Not to be confused with the `Jetpack_Plans` class (in `_inc/lib/plans.php`), which
  * fetches general information about all available plans from WordPress.com, side-effect free.
  *
  * @package automattic/jetpack
  */
 
-use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Current_Plan;
 
 /**
  * Provides methods methods for fetching the site's plan and products from WordPress.com.
  */
 class Jetpack_Plan {
-	/**
-	 * A cache variable to hold the active plan for the current request.
-	 *
-	 * @var array
-	 */
-	private static $active_plan_cache;
-
 	/**
 	 * The name of the option that will store the site's plan.
 	 *
+	 * @deprecated $$next-version$$ use Automattic\Jetpack\Current_Plan::PLAN_OPTION
+	 *
 	 * @var string
 	 */
 	const PLAN_OPTION = 'jetpack_active_plan';
```

Comment:
Worth doing like this?
```suggestion
	const PLAN_OPTION = Current_Plan::PLAN_OPTION;
```
and the same for the other constants below?

Reply:
Sure thing! b7ad7fc23979d88a0d7da164219b094cb677b47e